src/resources/views/manual/tutorials/qgis-getting-started.blade.php

```diff
@@ -0,0 +1,69 @@
+@extends('manual.base')
+
+@section('manual-title', 'Getting Started')
+
+@section('manual-content')
+  <div class="row">
+    <p class="lead">Introduction on Interfacing Biigle Api with QGis.</p>
+    <p>
+      QGis is an open source Geographic Information System that is used for viewing, editing and analysis of geospatial data. In order to interface Biigle with QGis, API endpoints, which respond with GeoJSON format Data, had been added. These API’s can only be accessed by Project Editor, Expert and Admins.
+    </p>
+
+    <p>
+      The GeoJSON Format data From the API can be imported to QGIS by creating a new Vector Layer in QGis. In the following examples you will learn how to Successfully configure the Authentication Credential for Biigle account in QGIS, add a Vector Layer and import Images data points of a volume whose id is specified in Api URI.
+    </p>
+    <p><i>Note: For this manual, QGIS Version 3.14 is being used.</i></p>
```

Comment:
You refer to version 3.14 here but all links to the QGIS manual point to version 3.10. Please update the links (or the version number here).